src/client/datascience/raw-kernel/rawJupyterSession.ts

```diff
@@ -122,19 +121,6 @@ export class RawJupyterSession extends BaseJupyterSession {
                 sendKernelTelemetryEvent(this.resource, Telemetry.RawKernelSessionStartTimeout);
                 traceError('Raw session failed to start in given timeout');
                 throw error;
-            } else if (error instanceof IpyKernelNotInstalledError) {
-                sendKernelTelemetryEvent(
-                    this.resource,
-                    Telemetry.RawKernelSessionStart,
-                    stopWatch.elapsedTime,
-                    undefined,
-                    error
-                );
-                sendKernelTelemetryEvent(this.resource, Telemetry.RawKernelSessionStartNoIpykernel, {
```

Comment:
We need this telemetry, couldn't find it being sent anywhere.

Reply:
Yeah you're right. I think I might have to send this higher up in the 'connectToKernel' function. The intent of this is to indicate somebody got the prompt, I think? At least the reason returns what choice they made.

Reply:
I moved this into the 'connectToKernel' function.